### ISSUE TITLE
File transfer very slow 

### ISSUE THREAD

**Issue by Abdull** [cowbell/sharedrop#91]
Both computers are on the same network. Transfer is slower than download from public Internet. Transferring 100 MB takes 30 seconds within local network.

**Comment by amirasyraf** [comment]
Yep it's slow. I'm only getting average of 27 Mbps. 27 Mbps is around 3.37 MB/s which is pretty much in line with what you're getting.
